3p/iframe-messaging-client.js

```diff
@@ -74,8 +76,8 @@ export class IframeMessagingClient {
    *  @param {Object=} opt_payload The payload of message to send.
    */
   sendMessage(type, opt_payload) {
-    this.hostWindow_.postMessage/*OK*/(
-        serializeMessage(type, this.sentinel_, opt_payload), '*');
+    this.hostWindow_.postMessage/*OK*/(serializeMessage(
+        type, this.sentinel_, opt_payload, this.rtvVersion_), '*');
```

Comment:
but that's because `this.rtvVersion_` can be `null`, not that one can choose to not send rtvVersion number.  Basically when `win.AMP_CONFIG.v` is not `null` we will always send message with rtv version number? 
Or maybe one time is ok, why we sent that same piece of info every time?